pkg/apis/flowcontrol/validation/validation_test.go

```diff
@@ -355,9 +361,10 @@ func TestFlowSchemaValidation(t *testing.T) {
 							},
 							ResourceRules: []flowcontrol.ResourcePolicyRule{
 								{
-									Verbs:     []string{flowcontrol.VerbAll},
-									APIGroups: []string{flowcontrol.APIGroupAll},
-									Resources: []string{flowcontrol.ResourceAll},
+									Verbs:      []string{flowcontrol.VerbAll},
+									APIGroups:  []string{flowcontrol.APIGroupAll},
+									Resources:  []string{flowcontrol.ResourceAll},
+									Namespaces: []string{flowcontrol.NamespaceEvery},
```

Comment:
We should have at least one positive test case for `flowcontrol.NamespaceClusterScope` by itself and one in combination with NamespaceEvery, I think?